drivers/include/periph/wdt.h

```diff
@@ -0,0 +1,290 @@
+/*
+ * Copyright (C) 2017 Technische Universität Berlin
+ *           (C) 2019 Inria
+ *           (C) 2019 ML!PA Consulting GmbH
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    drivers_periph_wdt WDT
+ * @ingroup     drivers_periph
+ * @brief       Watchdog timer peripheral driver
+ *
+ * A watchdog timer (WDT) is an electronic or software timer that is
+ * used to detect and recover from unusual or suspect behaviour as well as
+ * software malfunctions. During `NORMAL` operation the application will reset
+ * wdt_kick() the timer counter preventing it from hitting 0. If due to
+ * software or hardware error the WDT counter isn't reset this will trigger
```

Comment:
Here I would replace `this` by `it` and maybe add a comma before `it` (`it` means the watchdog in this case.).